sdk/identity/Azure.Identity.Broker/tests/DefaultAzureCredentialFactoryTests.cs

```diff
@@ -56,8 +57,8 @@ public void ValidateDefaultAzureCredentialAZURE_TOKEN_CREDENTIALS_Honored_WithBr
                     Assert.IsTrue(chain.Any(cred => cred is AzurePowerShellCredential));
                     Assert.IsTrue(chain.Any(cred => cred is VisualStudioCredential));
                     Assert.IsTrue(chain.Any(cred => cred is AzureDeveloperCliCredential));
-                    Assert.IsTrue(chain.Any(cred => cred.GetType() == typeof(VisualStudioCodeCredential)));
-                    Assert.IsTrue(chain.Any(cred => cred.GetType() == typeof(InteractiveBrowserCredential)));
+                    Assert.IsTrue(chain.Any(cred => cred is VisualStudioCodeCredential));
+                    Assert.IsTrue(chain.Any(cred => cred is BrokerCredential));
```

Comment:
Are there other tests in here that should be updated for BrokerCredential? For example, `ValidateExcludeOptionsHonoredWithAZURE_TOKEN_CREDENTIALS_DevMode`.

Reply:
Yes